_posts/2021-2-11-Overview-of-Today's-Work.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Overview of Today's Work
+description: 
+summary: 
+tags: 
+---
+Our starting task for today was to continue progressing our typing accuracy and speed on Typing club. Once we finished, Kajari gave us a link to a website to practice and introduce us to code we will use to create a small video game. We first created a new folder and added the html elemnts onto the folder named breakout. The practice consisted of 10 chapters, where each chapter focused on creating certain elements and functions for the small ball point game. After completing the practice, we made a new repository named after the file, and pushed up the repository through terminal to pop up on our github.io's. After pushing up the repository, we invited Kajari as a collaborator and sent her the link to our completed practice game. [Here is the completed game](https://osvaldo178.github.io/breakout/).
```

Comment:
`elemnts` -> `elements`

Reply:
Please link to the article you used that stepped you through pushing the folder from your computer onto github.

Also please link to the article you used to create the game.